src/main/resources/Programming_Languages_Extensions.json

```diff
@@ -0,0 +1,3377 @@
+         ".xsp.metadata"
+      ]
+   },
+   {
+      "name":"XProc",
+      "type":"programming",
+      "extensions":[
+         ".xpl",
+         ".xproc"
+      ]
+   },
+   {
+      "name":"XQuery",
+      "type":"programming",
+      "extensions":[
+         ".xquery",
+         ".xq",
+         ".xql",
+         ".xqm",
+         ".xqy"
+      ]
+   },
+   {
+      "name":"XS",
+      "type":"programming",
+      "extensions":[
+         ".xs"
+      ]
+   },
+   {
+      "name":"XSLT",
+      "type":"programming",
+      "extensions":[
+         ".xslt",
+         ".xsl"
+      ]
+   },
+   {
+      "name":"Xojo",
+      "type":"programming",
+      "extensions":[
+         ".xojo_code",
+         ".xojo_menu",
+         ".xojo_report",
+         ".xojo_script",
+         ".xojo_toolbar",
+         ".xojo_window"
+      ]
+   },
+   {
+      "name":"Xtend",
+      "type":"programming",
+      "extensions":[
+         ".xtend"
+      ]
+   },
+   {
+      "name":"YAML",
+      "type":"data",
+      "extensions":[
+         ".yml",
+         ".reek",
+         ".rviz",
+         ".sublime-syntax",
+         ".syntax",
+         ".yaml",
+         ".yaml-tmlanguage"
+      ]
+   },
+   {
+      "name":"YANG",
+      "type":"data",
+      "extensions":[
+         ".yang"
+      ]
+   },
+   {
+      "name":"Yacc",
+      "type":"programming",
+      "extensions":[
+         ".y",
+         ".yacc",
+         ".yy"
+      ]
+   },
+   {
+      "name":"Zephir",
+      "type":"programming",
+      "extensions":[
+         ".zep"
+      ]
+   },
+   {
+      "name":"Zimpl",
+      "type":"programming",
+      "extensions":[
+         ".zimpl",
+         ".zmpl",
+         ".zpl"
+      ]
+   },
+   {
+      "name":"desktop",
+      "type":"data",
+      "extensions":[
+         ".desktop",
+         ".desktop.in"
+      ]
+   },
+   {
+      "name":"eC",
+      "type":"programming",
+      "extensions":[
+         ".ec",
+         ".eh"
+      ]
+   },
+   {
+      "name":"edn",
+      "type":"data",
+      "extensions":[
+         ".edn"
+      ]
+   },
+   {
+      "name":"fish",
+      "type":"programming",
+      "extensions":[
+         ".fish"
+      ]
+   },
+   {
+      "name":"mupad",
+      "type":"programming",
+      "extensions":[
+         ".mu"
+      ]
+   },
+   {
+      "name":"nesC",
+      "type":"programming",
+      "extensions":[
+         ".nc"
+      ]
+   },
+   {
+      "name":"ooc",
+      "type":"programming",
+      "extensions":[
+         ".ooc"
+      ]
+   },
+   {
+      "name":"reStructuredText",
+      "type":"prose",
+      "extensions":[
+         ".rst",
+         ".rest",
+         ".rest.txt",
+         ".rst.txt"
+      ]
+   },
+   {
+      "name":"wisp",
+      "type":"programming",
+      "extensions":[
+         ".wisp"
+      ]
+   },
+   {
+      "name":"xBase",
+      "type":"programming",
+      "extensions":[
+         ".prg",
+         ".ch",
+         ".prw"
+      ]
+   }
+]
```

Comment:
Is this something we need to maintain? Can this be generated/downloaded from somewhere else?